example/config.ts

```diff
@@ -70,3 +91,46 @@ export const NON_TILED_LAYERS_EXAMPLES: NonTiledConfig[] = [
     },
   },
 ]
+
+export const WFS_LAYER_EXAMPLES: WfsConfig[] = [
+  {
+    title: 'Verblijfsobjecten (max level 13)',
+    id: 'verblijfsobject',
+    url: `${MAP_API_ROOT}maps/bag?REQUEST=Getfeature&VERSION=1.1.0&SERVICE=wfs&TYPENAME=ms:verblijfsobject&srsName=EPSG:4326&outputformat=geojson`,
+    options: {
+      onEachFeature: (feature, layer) => {
```

Comment:
What exactly is a feature compared to a layer?

Reply:
You can see the feature as the data and a leaflet-layer the `container` that presents thie location (or spatial form) of this data in the map. In leaflet, the layer concept is applied to all items that are presented on the the map. I find [this answer](https://stackoverflow.com/questions/10425553/definition-of-a-leaflet-layer) pretty clear. In the GIS slang, a layer refers to a different concept, that's why is so confusing.